src/components/ItemForm.js

```diff
@@ -23,58 +23,54 @@ const renderGeoField = ({ input, label, type, meta: { touched, error } }) => {
 }
 
 // to avoid the warnings from react about unnecessary props
-const renderTextField = field => <TextField {...field.input}/>
+const renderTextField = field => <TextField hintText={field.placeholder}  {...field.input}/>
 
 class ItemForm extends Component {
   render() {
     const { handleSubmit } = this.props
     return (
       <Container>
-        <Form className="quick-submit" onSubmit={handleSubmit} >
-          <Row>
-            <Col xs={{ size: 10, offset: 1 }}>
-              <h2>Add {this.props.itemType === 'Organization' ? 'an' : 'a'} {this.props.itemType}</h2>
-            </Col>
-          </Row>
+        <Form className='quick-submit' onSubmit={handleSubmit} >
+          <h2 className='form-title'>Add {this.props.itemType === 'organization' ? 'an' : 'a'} {this.props.itemType}</h2>
           <FormGroup>
             <Label>Title</Label>
             <Field name='title' className='title' placeholder='Neighbourhood Safety Jam' component={renderTextField} type='text'/>
           </FormGroup>
           <FormGroup>
             <Label>Summary</Label>
-            <Field name='summary' className='title' placeholder='Summary placeholder' component={renderTextField} type='text'/>
+            <Field name='summary' className='title' placeholder='Placeholder for summary ' component={renderTextField} type='text'/>
           </FormGroup>
           <Row className='imgField'>
-            <Col xs='2' className='pr-0'>
-              <img src={imgIcon} alt=""/>
+            <Col xs={2} sm={1}>
```

Comment:
Ah, gotcha. My last project was using an old version of Bootstrap, they looked kind of familiar, but not quite. Thanks!